nilearn/input_data/multi_nifti_masker.py

```diff
@@ -267,16 +270,19 @@ def transform_imgs(self, imgs_list, confounds=None, copy=True, n_jobs=1):
                                     'copy'])
 
         func = self._cache(filter_and_mask,
-                          ignore=['verbose', 'memory', 'memory_level', 'copy'])
+                           ignore=['verbose', 'memory', 'memory_level',
+                                   'copy'],
+                           shelve=self._shelving)
         data = Parallel(n_jobs=n_jobs)(
             delayed(func)(imgs, self.mask_img_, params,
-                           memory_level=self.memory_level,
-                           memory=self.memory,
-                           verbose=self.verbose,
-                           confounds=cfs,
-                           copy=copy)
+                          memory_level=self.memory_level,
+                          memory=self.memory,
+                          verbose=self.verbose,
+                          confounds=cfs,
+                          copy=copy,
+                          return_affine=False)
             for imgs, cfs in izip(niimg_iter, confounds))
-        return [d[0] for d in data]
+        return data
```

Comment:
Looking at this, it seems that the signature of the method has changed. Is that the case?